.idea/tech-circle.iml

```diff
@@ -77,6 +77,7 @@
     <orderEntry type="library" scope="PROVIDED" name="orm_adapter (v0.5.0, rbenv: 2.7.3) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="public_suffix (v4.0.6, rbenv: 2.7.3) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="puma (v5.4.0, rbenv: 2.7.3) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="pundit (v2.1.0, rbenv: 2.7.3) [gem]" level="application" />
```

Comment:
Is this RubyMine artifact and directory something that is useful to have stored in the repo? If not, I wonder if we should add it to the ignores to keep the diffs more focused.